include/CL/sycl/device.hpp

```diff
@@ -245,7 +245,7 @@ class device
 
 template <>
 size_t device::get_info<info::device::max_work_group_size>() const {
```

Comment:
I think this should be "info::device::max_compute_units" if we want to restrict for simple local accessor for now, i.e. with only one work-group.